### ISSUE TITLE
Remove duplicate non-deleted stamps from ceramic cache

### ISSUE THREAD

**Issue by nutrina** [gitcoinco/passport#2119]
This is supposed to be covered by migrations:
- 0018 -> remove duplicates
- 0019 -> add db constraints to avoid duplicates in the future

Unfortunately however, new duplicates are created after migration 18 and before migration 19 can execute, hence this fix cannot be deployed.

Potential solutions:
1. down time -> we turn everything off so that no new stamps are created and we can run the 2 migrations
2. use another temporary column for the constraints:
    - phase 1:
      a. add another column "provider_2" which has the same uniqness constraints
      b. remove the constraints for column provider
      c. save the same value in provider_2 as in provider
    - phase 2:
      -  after phase 1 is in prod, delete duplicates in the background
    - phase 3
      -  drop column provider_2
      - keep only provider, and apply the constraints ...

**Comment by nutrina** [comment]
This is duplicate of https://github.com/gitcoinco/passport/issues/2110
